mod/groups/start.php

```diff
@@ -329,7 +329,7 @@ function groups_set_icon_url($hook, $type, $url, $params) {
 		return "groupicon/$group->guid/$size/$icontime.jpg";
 	}
 
-	return "mod/groups/graphics/default{$size}.gif";
```

Comment:
Suppose a plugin points to

/mod/groups/graphics/defaultlarge.gif

In 2.0, a theme tries to override this new view we've created. The old image will still show in some cases because the plugin is pointing directly to the filesystem location.